client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -115,14 +122,14 @@ object FlightTableComponents {
     }
   }
 
-  def dateStringAsLocalDisplay(dt: String): String = dt match {
-    case "" => ""
-    case some => SDate.parse(dt).toLocalDateTimeString()
+  def dateStringAsLocalDisplay(dt: Option[MillisSinceEpoch]): String = dt match {
+    case None| Some(0) => ""
```

Comment:
We shouldn't get Some(0) - we should rather set this to None if we get a 0 in this field.